ansible/roles/loop_manage_cluster/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- name: Copy hub kubeconfig to target VM
+  copy:
+    src: /tmp/hub.kubeconfig
```

Comment:
Just checking, but a requirement to use this would be to have the hub cluster kubeconfig in your local /tmp directory?

Reply:
Ah, yes, I should be consistent. Will copy from {{ hub_cluster_kubeconfig }} to the remote system.